app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -603,6 +688,17 @@ def save(self, *args, **kwargs):
             return experiment
 
 
+class NimbusBranchScreenshotReadyForReviewSerializer(NimbusBranchScreenshotSerializer):
+    # Round-trip serialization & validation for review can use a string path
+    image = serializers.CharField(required=True)
+
+
+class NimbusBranchReadyForReviewSerializer(NimbusBranchSerializer):
+    screenshots = NimbusBranchScreenshotReadyForReviewSerializer(
+        many=True, required=False
+    )
```

Comment:
This only triggers when there are actually screenshots and then only to ensure that the screenshots have descriptions & images. 

Needed to add this to address a bunch of failing tests that fell over when screenshots *were* defined (e.g. via factory) and the validated data was a string rather than an image blob - e.g. [this round-trip pattern](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/tests/api/v5/test_serializers.py#L1735-L1742):

```python
        serializer = NimbusReadyForReviewSerializer(
            experiment,
            data=NimbusReadyForReviewSerializer(
                experiment,
                context={"user": self.user},
            ).data,
            context={"user": self.user},
        )
        self.assertTrue(serializer.is_valid())
```